problems/objects.js

```diff
@@ -61,7 +71,10 @@ const appleSet = () => {};
  *
  */
 
-const adamAndEveApples = () => {};
+const adamAndEveApples = (appleCountByName) => {
+  let sum = appleCountByName.Eve + appleCountByName.Adam;
```

Comment:
You don't necessarily need this sum variable to return the number. Can you refactor this function to one line.